storage/vdi_automaton_test.ml

```diff
@@ -0,0 +1,37 @@
+(*
+ * Copyright Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+open Vdi_automaton
+
+(* For any state [s] and operation [o] where [s' = s + o],
+   [if s <> s' then s - s' = op] *)
+
+let all_pairs x y =
+  List.fold_left (fun acc x -> List.map (fun y -> x, y) y @ acc) [] x
+
+let () =
+  List.iter
+    (fun (s, op) ->
+       try
+         let s' = s + op in
```

Comment:
Operator `+` and `-` are redefined inside Vdi_automaton, might make it clearer if this was `Vdi_automaton.(s + op)`. (or use some other `+..` symbol for it but that would require more invasive changes)[

Reply:
I think it's worth emphasizing it as you suggest. If Travis is happy I'll merge with the change